rsocket/RSocketServer.cpp

```diff
@@ -128,7 +128,10 @@ void RSocketServer::onRSocketSetup(
     throw result.error();
   }
   auto connectionParams = result.value();
-  CHECK(connectionParams.responder);
+  if (!connectionParams.responder) {
+    LOG(ERROR) << "Received invalid Responder. Dropping connection";
```

Comment:
Right, I wanted to treat the cases where the server-application returns a NULL responder as a error (which I saw in one of the tests).  And wanted to surface the error prominently in the logs.  It instead should be using RSocketException.